documentation/tutorial.md

```diff
@@ -238,3 +244,451 @@ You can verify that you added the user correctly, by entering the command `show
+```
+This should output something like:
+```
+​​unit-database-integrator-0:
+  UnitId: database-integrator/0
+  id: "24"
+  results:
+    mongodb:
+      database: mongodb
+      endpoints: 10.23.62.55,10.23.62.156
+      password: VMnRws6BlojzDi5e1m2GVWOgJaoSs44d
+      replset: mongodb
+      uris: mongodb://relation-4:VMnRws6BlojzDi5e1m2GVWOgJaoSs44d@10.23.62.55,10.23.62.156/mongodb?replicaSet=mongodb&authSource=admin
+      username: relation-4
+    ok: "True"
+  status: completed
+  timing:
+    completed: 2022-12-06 10:33:24 +0000 UTC
+    enqueued: 2022-12-06 10:33:20 +0000 UTC
+    started: 2022-12-06 10:33:24 +0000 UTC
+```
+*Note: your hostnames, usernames, and passwords will likely be different.*
+
+### Accessing the related database:
+Notice that in the previous step when you typed `juju run-action  database-integrator/0 get-credentials --wait` the command not only outputted the username, password, and database, but also outputted the URI. This means you do not have to generate the URI yourself. To connect to the database associated with the Database Integrator Charm enter:
+```
+mongosh "<uri from juju run-action  database-integrator/0 get-credentials --wait>"
+```
+*Note: be sure you wrap the URI in `"`*
+
+You will now be in the mongo shell as a non-admin user. Since you do not have admin access, privileged commands like `show users` should output:
+```
+MongoServerError: not authorized on mongodb to execute command { usersInfo: 1, lsid: { id: UUID("86d865cc-1f51-4b1e-8906-ea8a5159f906") }, $clusterTime: { clusterTime: Timestamp(1670340314, 1), signature: { hash: BinData(0, 25E321CD816AB02C5E92E2074A067CD33C4F0794), keyId: 7173987185652137989 } }, $db: "mongodb" }
+```
+Now exit the shell by typing `exit`
+
+### Removing the user
+To remove the user, remove the relation. Removing the relation automatically removes the user that was created when the relation was created. Enter the following to remove the relation:
+```
+juju remove-relation mongodb database-integrator
+```
+
+Now try again to connect to the same URI you just used in [Accessing the related database](#accessing-the-related-database):
+```
+mongosh "<uri from juju run-action  database-integrator/0 get-credentials --wait>"
+```
+
+This will output an error message:
+```
+Current Mongosh Log ID:	638f5ffbdbd9ec94c2e58456
+Connecting to:		mongodb://<credentials>@10.23.62.38,10.23.62.219/mongodb?replicaSet=mongodb&authSource=admin&appName=mongosh+1.6.1
+MongoServerError: Authentication failed.
+```
+As this user no longer exists. This is expected as `juju remove-relation mongodb database-integrator` also removes the user. 
+
+
+## Transcript Layer Security (TLS)
+
+TLS is the encryption of data sent between two applications. TLS requires minimal work to be set up on Charmed MongoDB. There already exists a [TLS Certificates Charm](https://charmhub.io/tls-certificates-operator) which handles providing, requesting, and renewing TLS certificates.
```

Comment:
amazing contribution here!! I will 1000% do this.

Minor thing here is I wouldn't use `Charmed Operator Framework` -- since this isn't the result of the Charmed Operator Framework (this is possible with the Reactive Charm Framework as well). I'll modify this and re-ping you :)